src/scheduled_tasks.rs

```diff
@@ -22,6 +22,7 @@ pub fn setup(pool: DbPool) -> Result<(), LemmyError> {
     active_counts(&conn);
     update_banned_when_expired(&conn);
     reindex_aggregates_tables(&conn, true);
+    drop_ccnew_indexes(&conn);
```

Comment:
I dont think this needs to be run every hour, but probably doesnt make a difference.